spec/requests/admin_timeout_spec.rb

```diff
@@ -8,6 +8,7 @@
       "provider" => "dfe",
       "info" => {"email" => "test-dfe-sign-in@host.tld"}
     )
+    allow_any_instance_of(DfeSignin::Authorisation).to receive(:has_role?) { true }
```

Comment:
Update `OmniAuth.config.mock_auth[:dfe]` above to contain the role rather than stubbing this code.